model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - id: metric.db.client.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.client.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
```

Comment:
would it make sense for this to have the same attributes as `db.client.operation.duration` (and `db.client.response.result_rows`)?

related, would it make sense for `db.client.operation.duration` (and `db.client.response.result_rows`) to have these two cosmos specific attributes in the cosmos case?
* `db.cosmosdb.sub_status_code`
* `db.cosmosdb.consistency_level`

Reply:
Here is, the list of dimensions mentioned, https://github.com/open-telemetry/semantic-conventions/blob/1daad9ce9b578e118754e39555eaa15ba1600ef7/docs/database/cosmosdb.md#metric-dbclientoperationduration

Reply:
unresolving this discussion to give @lmolkova a chance to review

Reply:
> Here is, the list of dimensions mentioned, https://github.com/open-telemetry/semantic-conventions/blob/1daad9ce9b578e118754e39555eaa15ba1600ef7/docs/database/cosmosdb.md#metric-dbclientoperationduration

oh, sorry, I totally missed:

> Refer [db.client.cosmosdb.operation.request_charge](https://github.com/open-telemetry/semantic-conventions/blob/1daad9ce9b578e118754e39555eaa15ba1600ef7/docs/database/cosmosdb.md#metric-dbclientcosmosdboperationrequest_charge) metrics for dimensions.